src/components/modules/banners/contributions/BannerTypes.ts

```diff
@@ -1,4 +1,4 @@
-import { TickerSettings } from '../lib/variants';
+import { TickerSettings } from '../../../../lib/variants';
```

Comment:
I've used https://github.com/tleunen/babel-plugin-module-resolver in the past to create aliases for module paths to avoid lots of directory navigation with imports. We're not using babel on this project but I wonder if there's an equivalent config for typescript. Another benefit is that if you move a file up or down the directory hierarchy you don't have to update all relative paths. This [blog post](https://medium.com/slackernoon/use-typescript-aliases-to-clean-up-your-import-statements-7210b7ec2af1) suggests it's possible to configure this in our `tsconfig.json`.